packages/zipkin-transport-kafka/test/integrationTest.js

```diff
@@ -112,4 +112,105 @@ describe('Kafka transport - integration test', () => {
       done(err);
     });
   });
+
+  it('should send trace data to Kafka without zookeeper', function(done) {
```

Comment:
This is copypasta except for `useZookeeper: false`. Might be prudent to make it DRYer?